rootfs/usr/local/etc/php/conf.d/30-xdebug.ini

```diff
@@ -0,0 +1,10 @@
+;zend_extension = xdebug.so
```

Comment:
Maybe we should find a way to load this ini just on dev, so we don't have to uncomment this

Reply:
Yup. For dev environments we mount a volume with specific configuration values.